docs/source/supporting/examples/ConfigActuation.rst

```diff
@@ -0,0 +1,23 @@
+.. _ConfigActuation:
+
+=========================
+Config Actuation Example
```

Comment:
I talked with @jhaack we are going with the single line under of = signs rather than both below and above for the title.  Please update this throughout your documentation.